INIT.MD

```diff
@@ -0,0 +1,42 @@
+How to initialize the docker containers:
+
+Make sure to create all of the folders that will be used by the volumes first. If you don't it creates weird permissions issues. Also create a file called `monstache.config.toml` in the monstache volume directory. It may be necessary to set `elasticsearch-healthcheck-timeout` and `elasticsearch-healthcheck-timeout-startup` to larger values in this config file depending on how slow your computer is.
+
+On Linux hosts run  `sysctl -w vm.max_map_count=262144`  and then `sysctl --system`
+
+See: https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html#docker-prod-cluster-composefile
+
+Change your working directory to the top level directory
+
+These commands initialize the replica set for mongo:
```

Comment:
I was hoping for a script that performed all of these for the user. But this is a good start.

Reply:
The problem here is that for some reason this stuff won't work within a script. There's an issue with it attempting to do stuff before the other service is ready.

Reply:
As in it will start up mongo, and then attempt to execute code on the mongo instance before mongo is fully running.